scenarioo-client/app/shared/metadata/tree.component.ts

```diff
@@ -30,27 +30,29 @@ function TreeComponentController($sce) {
     const ITEM = 'Item';
     const CHILDREN = 'children';
 
+    console.log('treeComponent: ' + scope.branchInformationTree);
+
     this.$onChanges = (changes) => {
-        if (changes.data) {
+        if (changes.branchInformationTree) {
             /* there is no 'ng-bind-html-unsafe' anymore. we use Strict Contextual Escaping, see
              http://docs.angularjs.org/api/ng/service/$sce for more information
              */
-            scope.treeHtml = $sce.trustAsHtml(createTreeHtml(changes.data.currentValue));
+            scope.treeHtml = $sce.trustAsHtml(createTreeHtml(changes.branchInformationTree.currentValue));
         }
     };
 
-    function createTreeHtml(data) {
+    function createTreeHtml(branchInformationTree) {
 
-        if (!angular.isObject(data)) {
-            return 'no data to display';
-        } else if (angular.isObject(data) && angular.isArray(data)) {
+        if (!angular.isObject(branchInformationTree)) {
+            return 'no branchInformationTree to display';
```

Comment:
is this something the user is going to see? Then I would prefer the term data as branchInformationTree is very technical